taipy/gui/_renderers/factory.py

```diff
@@ -153,6 +154,25 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "time": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="TimeSelector",
+            attributes=attrs,
+            default_value=datetime.fromtimestamp(0),
```

Comment:
would now be better ?

Reply:
I'm not sure what u asked for at this comment

Reply:
just saying that `=datetime.today()` might more interesting

Reply:
or even better `datetime.today().time()`

Reply:
Agree, done